src/main/java/org/repodriller/scm/GitRemoteRepository.java

```diff
@@ -1,163 +1,167 @@
-		return FileUtils.getTempDirectory().getAbsolutePath();
-	}
-
-	protected static String gitRemoteRepositoryTempDir(String remoteRepositoryUrl, String rootTempDir) {
-		int lastIndexOfDotGit = remoteRepositoryUrl.lastIndexOf(".git");
-		if(lastIndexOfDotGit < 0 )
-			lastIndexOfDotGit = remoteRepositoryUrl.length();
-		String directoryName = remoteRepositoryUrl.substring(remoteRepositoryUrl.lastIndexOf("/")+1, lastIndexOfDotGit);
-
-		if(!rootTempDir.endsWith(File.separator))
-			rootTempDir += File.separator;
-
-		return rootTempDir + directoryName;
-	}
-
-	public static SCMRepository singleProject(String url) {
-		return singleProject(url, gitSystemTempDir(), false);
-	}
-
-	protected static SCMRepository singleProject(String url, String rootTempGitPath, boolean bare) {
-		return new GitRemoteRepository(url, rootTempGitPath, bare).info();
-	}
-
-	public static SCMRepository[] allProjectsIn(List<String> urls) {
-		return allProjectsIn(urls, gitSystemTempDir(), false);
-	}
-
-	protected static SCMRepository[] allProjectsIn(List<String> urls, String rootTempGitPath, boolean bare) {
-		List<SCMRepository> repos = new ArrayList<SCMRepository>();
-		for (String url : urls) {
-			repos.add(singleProject(url, rootTempGitPath, bare));
-		}
-
-		return repos.toArray(new SCMRepository[repos.size()]);
-	}
-
-	public void deleteTempGitPath() throws IOException {
-		FileUtils.deleteDirectory(new File(this.tempGitPath));
-	}
-
-	@Override
-	public SCMRepository info() {
-		return tempGitRepository.info();
-	}
-
-	@Override
-	public ChangeSet getHead() {
-		return tempGitRepository.getHead();
-	}
-
-	@Override
-	public List<ChangeSet> getChangeSets() {
-		return tempGitRepository.getChangeSets();
-	}
-
-	@Override
-	public Commit getCommit(String id) {
-		return tempGitRepository.getCommit(id);
-	}
-
-	@Override
-	public void checkout(String hash) {
-		tempGitRepository.checkout(hash);
-	}
-
-	@Override
-	public List<RepositoryFile> files() {
-		return tempGitRepository.files();
-	}
-
-	@Override
-	public void reset() {
-		tempGitRepository.reset();
-	}
-
-	@Override
-	public long totalCommits() {
-		return tempGitRepository.totalCommits();
-	}
-
-	@Override
-	@Deprecated
-	public String blame(String file, String currentCommit, Integer line) {
-		return tempGitRepository.blame(file, currentCommit, line);
-	}
-
-	@Override
-	public List<BlamedLine> blame(String file, String commitToBeBlamed, boolean priorCommit) {
-		return tempGitRepository.blame(file, commitToBeBlamed, priorCommit);
-	}
-
-	public static SingleGitRemoteRepositoryBuilder hostedOn(String gitUrl) {
-		return new SingleGitRemoteRepositoryBuilder(gitUrl);
-	}
-
-	public static MultipleGitRemoteRepositoryBuilder hostedOn(List<String> gitUrls) {
-		return new MultipleGitRemoteRepositoryBuilder(gitUrls);
-	}
-
-	@Override
-	public String getCommitFromTag(String tag) {
-		return tempGitRepository.getCommitFromTag(tag);
-	}
-
-}
+package org.repodriller.scm;

```

Comment:
Why is this entire file changed?

Reply:
Hmm, I think some kind of spacing issues. Let me see what I can do about it.